src/imitation/algorithms/bc.py

```diff
@@ -431,11 +416,17 @@ def _on_epoch_end(epoch_number: int):
             if on_epoch_end is not None:
                 on_epoch_end()
 
+        n_minibatches = (
```

Comment:
This is a bit long for one expression, can we split it up more? E.g. introduce a temporary variable `num_minibatches = self.batch_size // self.minibatch_size`

Reply:
I don't think there's any point, since it wouldn't be used elsewhere, and the verbosity is due to checking that `n_batches` is not None.